app/src/behaviors/behavior_transparent.c

```diff
@@ -14,8 +14,6 @@
 
 LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
-#if DT_HAS_COMPAT_STATUS_OKAY(DT_DRV_COMPAT)
```

Comment:
Should `DT_DRV_COMPAT` defines also be cleaned up?

Reply:
No, because `BEHAVIOR_DT_INST_DEFINE` uses it